### PR TITLE
✨ Add type adyoulike for the amp-ad tag 

### PR DESCRIPTION
**Pull request by j343my**
Adding Adyoulike implementation of amp-ad to amphtml

### CODE REVIEW

**Comment by j343my** [comment]
I signed it!

File: ads/_config.js [hunk 1]
```diff
@@ -252,6 +252,11 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'adyoulike': {
+    consentHandlingOverride: true,
+    prefetch: 'https://pixels.omnitagjs.com/amp.js',
```
Comment (lannka) [inline ads/_config.js]: could you pls follow [this guideline](https://github.com/ampproject/amphtml/blob/master/ads/README.md#other-tips) to implement render-start API?

Comment (j343my) [reply ads/_config.js]: done :)

File: ads/adyoulike.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Serves ads from [Adyoulike](https://www.adyoulike.com/).
+
+## Example
+
+### Normal ad
+
+```html
+<amp-ad width="300" height="250"
+      type="adyoulike"
+      data-placement=""
```
Comment (calebcordry) [inline ads/adyoulike.md]: Could you explain why this is required if it is an empty value?

Comment (calebcordry) [reply ads/adyoulike.md]: Actually it seems as this is only for this example, but is a bit confusing. Can you put a fake value here?

Comment (j343my) [reply ads/adyoulike.md]: Yes it's only for this example. I put a value

Comment (calebcordry) [reply ads/adyoulike.md]: Thanks!

**Comment by calebcordry** [comment]
LGTM thanks. Looks like travis is failing here, I will restart but you might need to rebase. Let me know if you see it fail/pass again before I do.

**Comment by j343my** [comment]
@calebcordry It seems Travis pass

**Comment by calebcordry** [comment]
Thanks for contributing!

**Pull request merged**